src/js/app.js

```diff
@@ -69,6 +74,10 @@ app.config(['$stateProvider', '$urlRouterProvider', '$locationProvider', '$httpP
 
     $urlRouterProvider.otherwise('/404');
 
+    ngMetaProvider.setDefaultTitle('Hourai Teahouse | ');
+    ngMetaProvider.useTitleSuffix(true);
+    ngMetaProvider.setDefaultTitleSuffix('Doujin Development');
+
     // State configuration
     $stateProvider
```

Comment:
We should probably set the state metadata in the `data` property to allow for inheritance of the tags on child states that choose not to override the values: https://github.com/vinaygopinath/ngMeta#data-inheritance-in-ui-router